src/main/python/covata/delta/apiclient.py

```diff
@@ -229,7 +225,7 @@ def get_secret(self, requestor_id, secret_id):
         :param str requestor_id: the authenticating identity id
         :param str secret_id: the secret id to be retrieved
         :return: the retrieved secret
-        :rtype: dict[str, any]
+        :rtype: dict[str, str]
```

Comment:
Nice, yeah the only type we can be sure of is the key's which is a str. All good man, thanks!